test/trusted_resources_test.go

```diff
@@ -112,9 +112,16 @@ spec:
   tasks:
   - name: task
     taskRef:
-      name: %s
+      resolver: cluster
```

Comment:
Use cluster resolve for e2e test, since we don't have these examples stored in remote storage, it's hard to test other resolvers.